### PR TITLE
Use LabelSelector in overview to associate pods to a service

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

File: assets/app/scripts/services/data.js [hunk 1]
```diff
@@ -561,9 +561,9 @@ angular.module('openshiftConsole')
 
   // TODO Possibly remove these from DataService
   DataService.prototype.objectsByAttribute = function(objects, attr, map, actions, secondaryAttr) {
-    for (var i = 0; i < objects.length; i++) {
-      _objectByAttribute(objects[i], attr, map, actions ? actions[i] : null, secondaryAttr);
-    }
+    angular.forEach(objects, function(obj) {
+      _objectByAttribute(obj, attr, map, actions ? actions[i] : null, secondaryAttr);
```
Comment (liggitt) [inline assets/app/scripts/services/data.js]: using `i` which doesn't exist now?

**Comment by liggitt** [comment]
LGTM

**Comment by jwforres** [comment]
[merge]

**Comment by jwforres** [comment]
unrelated [merge]

**Pull request merged**